### PR TITLE
fix: removed role shortgroup

### PR DESCRIPTION
**Pull request by Kerwood**
Signed-off-by: Patrick Kerwood <patrick@kerwood.dk>

### Description of your changes
Removes the short group for the Role resource, since there are no other resources in that group.
https://registry.terraform.io/providers/DataDog/datadog/latest/docs/resources/role

Before
```
roles.role.datadog.jet.crossplane.io
```
After
```
roles.datadog.jet.crossplane.io
```

I have:

- [x] Read and followed Crossplane's [contribution process].
- [x] Run `make reviewable test` to ensure this PR is ready for review.

### CODE REVIEW

File: internal/controller/zz_setup.go [hunk 1]
```diff
@@ -21,25 +21,25 @@ import (
...
 // the supplied manager.
 func Setup(mgr ctrl.Manager, o controller.Options) error {
 	for _, setup := range []func(ctrl.Manager, controller.Options) error{
+		role.Setup,
 		downtime.Setup,
 		json.Setup,
 		monitor.Setup,
 		servicelevelobjective.Setup,
 		providerconfig.Setup,
-		role.Setup,
```
Comment (ytsarev) [inline internal/controller/zz_setup.go:44]: Nit: why change the alphabetic order here?

**Comment by ytsarev** [comment]
Please also resolve the conflicts

File: internal/controller/zz_setup.go (hunk 1 above)
Comment (Kerwood) [reply internal/controller/zz_setup.go:44]: Didn't think about alphabetic order, I just appended :)

Comment (Kerwood) [reply internal/controller/zz_setup.go:44]: No, wait. That file is auto generated. I don't control that ?

Comment (Kerwood) [reply internal/controller/zz_setup.go:44]: Maybe it lists resources with no short group first ?

**Comment by Kerwood** [comment]
> Please also resolve the conflicts

Fixed..

**Review by ytsarev — APPROVED** [review]

**Pull request merged**